### ISSUE TITLE
New submissions for Fri,  2 Dec 22

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#22]
## Keyword: volume render
There is no result 
### NeuWigs: A Neural Dynamic Model for Volumetric Hair Capture and  Animation
 - **Authors:** Ziyan Wang, Giljoo Nam, Tuur Stuyck, Stephen Lombardi, Chen Cao, Jason Saragih, Michael Zollhoefer, Jessica Hodgins, Christoph Lassner
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Graphics (cs.GR)
 - **Arxiv link:** https://arxiv.org/abs/2212.00613
 - **Pdf link:** https://arxiv.org/pdf/2212.00613
 - **Abstract**
 The capture and animation of human hair are two of the major challenges in the creation of realistic avatars for the virtual reality. Both problems are highly challenging, because hair has complex geometry and appearance, as well as exhibits challenging motion. In this paper, we present a two-stage approach that models hair independently from the head to address these challenges in a data-driven manner. The first stage, state compression, learns a low-dimensional latent space of 3D hair states containing motion and appearance, via a novel autoencoder-as-a-tracker strategy. To better disentangle the hair and head in appearance learning, we employ multi-view hair segmentation masks in combination with a differentiable volumetric renderer. The second stage learns a novel hair dynamics model that performs temporal hair transfer based on the discovered latent codes. To enforce higher stability while driving our dynamics model, we employ the 3D point-cloud autoencoder from the compression stage for de-noising of the hair state. Our model outperforms the state of the art in novel view synthesis and is capable of creating novel hair animations without having to rely on hair observations as a driving signal.
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imagingremote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
### AUG-FedPrompt: Practical Few-shot Federated NLP with Data-augmented  Prompts
 - **Authors:** Dongqi Cai, Yaozong Wu, Haitao Yuan, Shangguang Wang, Felix Xiaozhu Lin, Mengwei Xu
 - **Subjects:** Computation and Language (cs.CL); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2212.00192
 - **Pdf link:** https://arxiv.org/pdf/2212.00192
 - **Abstract**
 Transformer-based pre-trained models have become the de-facto solution for NLP tasks. Fine-tuning such pre-trained models for downstream tasks often requires tremendous amount of data that is both private and labeled. However, in reality: 1) such private data cannot be collected and is distributed across mobile devices, and 2) well-curated labeled data is scarce. To tackle those issues, we first define a data generator for federated few-shot learning tasks, which encompasses the quantity and distribution of scarce labeled data in a realistic setting. Then we propose AUG-FedPrompt, a prompt-based federated learning algorithm that carefully annotates abundant unlabeled data for data augmentation. AUG-FedPrompt can perform on par with full-set fine-tuning with very few initial labeled data.
## Keyword: video retrieval
There is no result 
### Smart Insole: A Gait Analysis Monitoring Platform Targeting Parkinson  Disease Patients Based on Insoles
 - **Authors:** Dimitrios Boucharas, Christos Androutsos, George Gkois, Vassilis Tsakanikas, Vasileios Pezoulas, Dimitrios Manousos, Vasileios Skaramagkas, Chariklia Chatzaki, Stathis Kontogiannis, Christos Spandonidis, Alexandros Pantazis, Nikolaos Tachos, Manolis Tsiknakis, Dimitrios Fotiadis
 - **Subjects:** Human-Computer Interaction (cs.HC)
 - **Arxiv link:** https://arxiv.org/abs/2212.00109
 - **Pdf link:** https://arxiv.org/pdf/2212.00109
 - **Abstract**
 During the preceding decades, human gait analysis has been the center of attention for the scientific community, while the association between gait analysis and overall health monitoring has been extensively reported. Technological advances further assisted in this alignment, resulting in access to inexpensive and remote healthcare services. Various assessment tools, such as software platforms and mobile applications, have been proposed by the scientific community and the market that employ sensors to monitor human gait for various purposes ranging from biomechanics to the progression of functional recovery. The framework presented herein offers a valuable digital biomarker for diagnosing and monitoring Parkinson's disease that can help clinical experts in the decision-making process leading to corrective planning or patient-specific treatment. More accurate and reliable decisions can be provided through a wide variety of integrated Artificial Intelligence algorithms and straightforward visualization techniques, including, but not limited to, heatmaps and bar plots. The framework consists of three core components: the insole pair, the mobile application, and the cloud-based platform. The insole pair deploys 16 plantar pressure sensors, an accelerometer, and a gyroscope to acquire gait data. The mobile application formulates the data for the cloud platform, which orchestrates the component interaction through the web application. Utilizing open communication protocols enables the straightforward replacement of one of the core components with a relative one (e.g., a different model of insoles), transparently from the end user, without affecting the overall architecture, resulting in a framework with the flexibility to adjust its modularity.
### AUG-FedPrompt: Practical Few-shot Federated NLP with Data-augmented  Prompts
 - **Authors:** Dongqi Cai, Yaozong Wu, Haitao Yuan, Shangguang Wang, Felix Xiaozhu Lin, Mengwei Xu
 - **Subjects:** Computation and Language (cs.CL); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2212.00192
 - **Pdf link:** https://arxiv.org/pdf/2212.00192
 - **Abstract**
 Transformer-based pre-trained models have become the de-facto solution for NLP tasks. Fine-tuning such pre-trained models for downstream tasks often requires tremendous amount of data that is both private and labeled. However, in reality: 1) such private data cannot be collected and is distributed across mobile devices, and 2) well-curated labeled data is scarce. To tackle those issues, we first define a data generator for federated few-shot learning tasks, which encompasses the quantity and distribution of scarce labeled data in a realistic setting. Then we propose AUG-FedPrompt, a prompt-based federated learning algorithm that carefully annotates abundant unlabeled data for data augmentation. AUG-FedPrompt can perform on par with full-set fine-tuning with very few initial labeled data.
### Efficient stereo matching on embedded GPUs with zero-means cross  correlation
 - **Authors:** Qiong Chang, Aolong Zha, Weimin Wang, Xin Liu, Masaki Onishi, Lei Lei, Meng Joo Er, Tsutomu Maruyama
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Hardware Architecture (cs.AR)
 - **Arxiv link:** https://arxiv.org/abs/2212.00476
 - **Pdf link:** https://arxiv.org/pdf/2212.00476
 - **Abstract**
 Mobile stereo-matching systems have become an important part of many applications, such as automated-driving vehicles and autonomous robots. Accurate stereo-matching methods usually lead to high computational complexity; however, mobile platforms have only limited hardware resources to keep their power consumption low; this makes it difficult to maintain both an acceptable processing speed and accuracy on mobile platforms. To resolve this trade-off, we herein propose a novel acceleration approach for the well-known zero-means normalized cross correlation (ZNCC) matching cost calculation algorithm on a Jetson Tx2 embedded GPU. In our method for accelerating ZNCC, target images are scanned in a zigzag fashion to efficiently reuse one pixel's computation for its neighboring pixels; this reduces the amount of data transmission and increases the utilization of on-chip registers, thus increasing the processing speed. As a result, our method is 2X faster than the traditional image scanning method, and 26% faster than the latest NCC method. By combining this technique with the domain transformation (DT) algorithm, our system show real-time processing speed of 32 fps, on a Jetson Tx2 GPU for 1,280x384 pixel images with a maximum disparity of 128. Additionally, the evaluation results on the KITTI 2015 benchmark show that our combined system is more accurate than the same algorithm combined with census by 7.26%, while maintaining almost the same processing speed.
### A Survey of Mobile Edge Computing for the Metaverse: Architectures,  Applications, and Challenges
 - **Authors:** Yitong Wang, Jun Zhao
 - **Subjects:** Networking and Internet Architecture (cs.NI); Computational Engineering, Finance, and Science (cs.CE); Social and Information Networks (cs.SI)
 - **Arxiv link:** https://arxiv.org/abs/2212.00481
 - **Pdf link:** https://arxiv.org/pdf/2212.00481
 - **Abstract**
 Metaverse is an emerging virtual universe where humans can have real-time interactions and solid social links like in the physical world, and it opens up a new era of Internet and interactions. In Metaverse, an immersive and photorealistic environment promotes social activities, including education, meetings, and shopping of digital avatars based on critical technologies, including 3D rendering, extended reality, digital twins, artificial intelligence, and Blockchain. However, the limitations of computation, storage, and energy resources restrict the development of Metaverse, and a series of system issues (e.g., latency, security, and battery-life) continue to arise. As a result, how to find corresponding measurements to mitigate unsatisfactory influences becomes the focus. Mobile edge computing (MEC) as a distributed computing paradigm offloads computation-intensive tasks to the edge of the network. It brings the resources as close as possible to the end devices, addressing the shortcomings mentioned above. In this paper, we propose a comprehensive survey of the MEC-based Metaverse. Particular emphasis is given to the technologies convergence, architectures, and application scenarios, e.g., BoundlessXR and CloudXR. Significantly, we introduce the potential future directions for developing Metaverse systems.
### Exploiting Kernel Compression on BNNs
 - **Authors:** Franyell Silfa, Jose Maria Arnau, Antonio González
 - **Subjects:** Hardware Architecture (cs.AR); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2212.00608
 - **Pdf link:** https://arxiv.org/pdf/2212.00608
 - **Abstract**
 Binary Neural Networks (BNNs) are showing tremendous success on realistic image classification tasks. Notably, their accuracy is similar to the state-of-the-art accuracy obtained by full-precision models tailored to edge devices. In this regard, BNNs are very amenable to edge devices since they employ 1-bit to store the inputs and weights, and thus, their storage requirements are low. Also, BNNs computations are mainly done using xnor and pop-counts operations which are implemented very efficiently using simple hardware structures. Nonetheless, supporting BNNs efficiently on mobile CPUs is far from trivial since their benefits are hindered by frequent memory accesses to load weights and inputs. In BNNs, a weight or an input is stored using one bit, and aiming to increase storage and computation efficiency, several of them are packed together as a sequence of bits. In this work, we observe that the number of unique sequences representing a set of weights is typically low. Also, we have seen that during the evaluation of a BNN layer, a small group of unique sequences is employed more frequently than others. Accordingly, we propose exploiting this observation by using Huffman Encoding to encode the bit sequences and then using an indirection table to decode them during the BNN evaluation. Also, we propose a clustering scheme to identify the most common sequences of bits and replace the less common ones with some similar common sequences. Hence, we decrease the storage requirements and memory accesses since common sequences are encoded with fewer bits. We extend a mobile CPU by adding a small hardware structure that can efficiently cache and decode the compressed sequence of bits. We evaluate our scheme using the ReAacNet model with the Imagenet dataset. Our experimental results show that our technique can reduce memory requirement by 1.32x and improve performance by 1.35x.
### Simultaneously Transmitting and Reflecting RIS-Aided Mobile Edge  Computing: Computation Rate Maximization
 - **Authors:** Zhenrong Liu, Zongze Li, Miaowen Wen, Yi Gong, Yik-Chung Wu
 - **Subjects:** Information Theory (cs.IT); Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2212.00697
 - **Pdf link:** https://arxiv.org/pdf/2212.00697
 - **Abstract**
 In this paper, the novel simultaneously transmitting and reflecting (STAR) reconfigurable intelligent surface (RIS), which enables full-space coverage on users located on both sides of the surface, is investigated in the multi-user mobile edge computing (MEC) system. A computation rate maximization problem is formulated via the joint design of the STAR-RIS phase shifts, reflection and transmission amplitude coefficients, the receive beamforming vectors at the access point, and the users' energy partition strategies for local computing and offloading. Two operating protocols of STAR-RIS, namely energy splitting (ES) and mode switching (MS) are studied. Based on DC programming and semidefinite relaxation, an iterative algorithm is proposed for the ES protocol to solve the formulated non-convex problem. Furthermore, the proposed algorithm is extended to solve the non-convex, non-continuous MS problems with binary amplitude coefficients. Simulation results show that the resultant STAR-RIS-aided MEC system significantly improves the computation rate compared to the baseline scheme with conventional reflect-only/transmit-only RIS.
## Keyword: medical volume data
There is no result 
## Keyword: volume data
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl2
There is no result 
### Mixed Neural Voxels for Fast Multi-view Video Synthesis
 - **Authors:** Feng Wang, Sinan Tan, Xinghang Li, Zeyue Tian, Huaping Liu
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2212.00190
 - **Pdf link:** https://arxiv.org/pdf/2212.00190
 - **Abstract**
 Synthesizing high-fidelity videos from real-world multi-view input is challenging because of the complexities of real-world environments and highly dynamic motions. Previous works based on neural radiance fields have demonstrated high-quality reconstructions of dynamic scenes. However, training such models on real-world scenes is time-consuming, usually taking days or weeks. In this paper, we present a novel method named MixVoxels to better represent the dynamic scenes with fast training speed and competitive rendering qualities. The proposed MixVoxels represents the 4D dynamic scenes as a mixture of static and dynamic voxels and processes them with different networks. In this way, the computation of the required modalities for static voxels can be processed by a lightweight model, which essentially reduces the amount of computation, especially for many daily dynamic scenes dominated by the static background. To separate the two kinds of voxels, we propose a novel variation field to estimate the temporal variance of each voxel. For the dynamic voxels, we design an inner-product time query method to efficiently query multiple time steps, which is essential to recover the high-dynamic motions. As a result, with 15 minutes of training for dynamic scenes with inputs of 300-frame videos, MixVoxels achieves better PSNR than previous methods. Codes and trained models are available at https://github.com/fengres/mixvoxels
### A Survey of Mobile Edge Computing for the Metaverse: Architectures,  Applications, and Challenges
 - **Authors:** Yitong Wang, Jun Zhao
 - **Subjects:** Networking and Internet Architecture (cs.NI); Computational Engineering, Finance, and Science (cs.CE); Social and Information Networks (cs.SI)
 - **Arxiv link:** https://arxiv.org/abs/2212.00481
 - **Pdf link:** https://arxiv.org/pdf/2212.00481
 - **Abstract**
 Metaverse is an emerging virtual universe where humans can have real-time interactions and solid social links like in the physical world, and it opens up a new era of Internet and interactions. In Metaverse, an immersive and photorealistic environment promotes social activities, including education, meetings, and shopping of digital avatars based on critical technologies, including 3D rendering, extended reality, digital twins, artificial intelligence, and Blockchain. However, the limitations of computation, storage, and energy resources restrict the development of Metaverse, and a series of system issues (e.g., latency, security, and battery-life) continue to arise. As a result, how to find corresponding measurements to mitigate unsatisfactory influences becomes the focus. Mobile edge computing (MEC) as a distributed computing paradigm offloads computation-intensive tasks to the edge of the network. It brings the resources as close as possible to the end devices, addressing the shortcomings mentioned above. In this paper, we propose a comprehensive survey of the MEC-based Metaverse. Particular emphasis is given to the technologies convergence, architectures, and application scenarios, e.g., BoundlessXR and CloudXR. Significantly, we introduce the potential future directions for developing Metaverse systems.
### One-shot recognition of any material anywhere using contrastive learning  with physics-based rendering
 - **Authors:** Manuel S. Drehwald (3), Sagi Eppel (1 and 2 and 4), Jolina Li (2 and 4), Han Hao (2), Alan Aspuru-Guzik (1 and 2) ((1) Vector institute, (2) University of Toronto, (3) Karlsruhe Institute of Technology, (4) Innoviz)
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2212.00648
 - **Pdf link:** https://arxiv.org/pdf/2212.00648
 - **Abstract**
 We present MatSim: a synthetic dataset, a benchmark, and a method for computer vision based recognition of similarities and transitions between materials and textures, focusing on identifying any material under any conditions using one or a few examples (one-shot learning). The visual recognition of materials is essential to everything from examining food while cooking to inspecting agriculture, chemistry, and industrial products. In this work, we utilize giant repositories used by computer graphics artists to generate a new CGI dataset for material similarity. We use physics-based rendering (PBR) repositories for visual material simulation, assign these materials random 3D objects, and render images with a vast range of backgrounds and illumination conditions (HDRI). We add a gradual transition between materials to support applications with a smooth transition between states (like gradually cooked food). We also render materials inside transparent containers to support beverage and chemistry lab use cases. We then train a contrastive learning network to generate a descriptor that identifies unfamiliar materials using a single image. We also present a new benchmark for a few-shot material recognition that contains a wide range of real-world examples, including the state of a chemical reaction, rotten/fresh fruits, states of food, different types of construction materials, types of ground, and many other use cases involving material states, transitions and subclasses. We show that a network trained on the MatSim synthetic dataset outperforms state-of-the-art models like Clip on the benchmark, despite being tested on material classes that were not seen during training. The dataset, benchmark, code and trained models are available online.
### SparseFusion: Distilling View-conditioned Diffusion for 3D  Reconstruction
 - **Authors:** Zhizhuo Zhou, Shubham Tulsiani
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Graphics (cs.GR)
 - **Arxiv link:** https://arxiv.org/abs/2212.00792
 - **Pdf link:** https://arxiv.org/pdf/2212.00792
 - **Abstract**
 We propose SparseFusion, a sparse view 3D reconstruction approach that unifies recent advances in neural rendering and probabilistic image generation. Existing approaches typically build on neural rendering with re-projected features but fail to generate unseen regions or handle uncertainty under large viewpoint changes. Alternate methods treat this as a (probabilistic) 2D synthesis task, and while they can generate plausible 2D images, they do not infer a consistent underlying 3D. However, we find that this trade-off between 3D consistency and probabilistic image generation does not need to exist. In fact, we show that geometric consistency and generative inference can be complementary in a mode-seeking behavior. By distilling a 3D consistent scene representation from a view-conditioned latent diffusion model, we are able to recover a plausible 3D representation whose renderings are both accurate and realistic. We evaluate our approach across 51 categories in the CO3D dataset and show that it outperforms existing methods, in both distortion and perception metrics, for sparse-view novel view synthesis.
